### ISSUE TITLE
[bug] HTML <template> element empty when serving bundled files

### ISSUE THREAD

**Issue by ptrcnull** [tauri-apps/tauri#4410]
### Describe the bug

When `build.devPath` is a path to a directory, `<template>` elements are empty despite having contents in the source files:

```
<!-- ... -->
<template>
	<tr>
		<td class="id"></td>
		<td class="desc"></td>
	</tr>
</template>
<script src="common.js"></script>
<!-- ... -->
```

![image](https://user-images.githubusercontent.com/26630168/174596972-f7f4acb1-9368-4647-9699-a8b1cd108541.png)

### Platform and versions

```shell
Environment
  › OS: Alpine Linux 3.16.0 X64
  › Node.js: 16.15.0
  › npm: 8.12.1
  › pnpm: Not installed!
  › yarn: 1.22.19
  › rustup: Not installed!
  › rustc: 1.60.0
  › cargo: 1.60.0
  › Rust toolchain: 

Packages
  › @tauri-apps/cli [NPM]: 1.0.0
  › @tauri-apps/api [NPM]: 1.0.1
  › tauri [RUST]: 1.0.0,
  › tauri-build [RUST]: 1.0.0,
  › tao [RUST]: 0.11.2,
  › wry [RUST]: 0.18.3,

App
  › build-type: bundle
  › CSP: unset
  › distDir: ../dist
  › devPath: ../dist

App directory structure
  ├─ dist
  ├─ node_modules
  ├─ .git
  └─ src-tauri
```

**Comment by lucasfernog** [comment]
Seems like webkit2gtk custom URI doesn't support templates :/ you'll need to use https://github.com/tauri-apps/tauri-plugin-localhost/

**Comment by alainschneble** [comment]
> Seems like webkit2gtk custom URI doesn't support templates :/ you'll need to use https://github.com/tauri-apps/tauri-plugin-localhost/

Not sure if webkit2gtk was the culprit here, unless I'm missing something. See https://github.com/tauri-apps/tauri/issues/6165 and proposed fix https://github.com/tauri-apps/tauri/pull/6166.

**Comment by FabianLars** [comment]
@AlainSchneble This issue was fixed in #5247 after we realized that it's _not_ a webkitgtk specific issue (3 months later). The dev server you fixed in your PR wasn't a thing back then.

**Comment by alainschneble** [comment]
> ter we realized that it's _not_ a webkitgtk specific issue (3 months later). The dev server you fixed in your PR wasn't a thing back then.

Ok thanks, now I see!
